git/attribs.go

```diff
@@ -97,6 +98,22 @@ func GetAttributePaths(workingDir, gitDir string) []AttributePath {
 	return paths
 }
 
+// GetAttributeFilter returns a list of entries in .gitattributes which are
+// configured with the filter=lfs attribute as a file path filter which
+// file paths can be matched against
+// workingDir is the root of the working copy
+// gitDir is the root of the git repo
+func GetAttributeFilter(workingDir, gitDir string) *filepathfilter.Filter {
+	paths := GetAttributePaths(workingDir, gitDir)
+	patterns := make([]filepathfilter.Pattern, len(paths))
+
+	for i, path := range paths {
+		patterns[i] = filepathfilter.NewPattern(path.Path)
```

Comment:
In case we want to change this later in a way that takes away the 1-1 correspondence between paths and patterns, we can minimize that diff by calling `append()` instead of using the slicing operator `[]`. In order to make that work as-expected, you'll want to change L108 to initialize `patterns` with a capacity of `len(paths)`, instead of a `len` of `len(paths)`.